protocol/x/clob/keeper/clob_pair.go

```diff
@@ -304,6 +304,42 @@ func (k Keeper) GetAllClobPairs(ctx sdk.Context) (list []types.ClobPair) {
 	return
 }
 
+// validateLiquidationAgainstClobPairStatus returns an error if placing the provided
+// liquidation order would conflict with the clob pair's current status.
+func (k Keeper) validateLiquidationAgainstClobPairStatus(
+	ctx sdk.Context,
+	liquidationOrder types.LiquidationOrder,
+) error {
+	clobPair, found := k.GetClobPair(ctx, liquidationOrder.GetClobPairId())
+	if !found {
+		return errorsmod.Wrapf(
+			types.ErrInvalidClob,
+			"Clob %v is not a valid clob",
+			liquidationOrder.GetClobPairId(),
+		)
+	}
+
+	if !types.IsSupportedClobPairStatus(clobPair.Status) {
+		panic(
+			fmt.Sprintf(
+				"validateLiquidationAgainstClobPairStatus: clob pair status %v is not supported",
+				clobPair.Status,
+			),
+		)
+	}
```

Comment:
Hmm for my understanding wouldn't this logic allow a liquidation in an `INITIALIZING` market?

Why would that be valid? Shouldn't liquidations only be valid for `ACTIVE` markets?

Reply:
will change this to only allow if status is active